cli/src/commands/auth/login.ts

```diff
@@ -0,0 +1,41 @@
+import prompts from 'prompts';
+import { BaseCommand } from '../../base.js';
+import { readAPIKeyFromFile, writeAPIKey } from '../../key.js';
+
+export default class Login extends BaseCommand {
+  static description = 'Authenticate with Xata';
+
+  static examples = [];
+
+  static flags = {};
+
+  static args = [];
+
+  async run(): Promise<void> {
+    const existingKey = await readAPIKeyFromFile();
+    if (existingKey) {
+      const result = await prompts({
+        type: 'confirm',
+        name: 'overwrite',
+        message: 'Authentication is already configured, do you want to overwrite it?'
+      });
+      if (!result.overwrite) this.exit(2);
```

Comment:
Are we documenting error codes somewhere?

Reply:
Good point. I'll add a note on the issue for improvements.